### DISCUSSION TITLE
Custom tariff

### DISCUSSION THREAD

**Discussion by kalaws** [goecharger/go-eCharger-API-v2#174] [Q&A]
Hello,

is there any way to get a custom tariff/list of hourly prices into the charger? I have a made a proprietary tariff where hours 9-17 are made to reflect feedin price rather than grid import price, in order to schedule charging more appropriately, but still without throwing in solar forecasting into the mix. 

If that is not possible, any way to push a charge schedule? OCPP?

**Comment by kalaws** [comment]
Or redirect the http request of the charger as it fetches awattar prices?

**Reply by povlhp** [reply]
Today my go-e is missing electricity prices. Reboot did not help. 

Please let us know how to manually feed it prices. Expose a writeable url and a data format.
